gitdash/components/card.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  Heading,
+  Box,
+  Center,
+  Text,
+  Stack,
+  Tag,
+  useColorModeValue,
+  Button,
+  Link,
+} from "@chakra-ui/react";
+
+function extractRepoName(issueUrl: string) {
+  var url = new URL(issueUrl);
+  var pathArray = url.pathname.split("/");
+  return pathArray[1] + "/" + pathArray[2];
```

Comment:
Using `+` works, although I tend to prefer using [template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) in JavaScript these days.